scripts/multi_component_hydrogen_bond_propensity/multi_component_hydrogen_bond_propensity_report.py

```diff
@@ -363,19 +363,30 @@ def main(structure, work_directory, library, csdrefcode):
 
 if __name__ == '__main__':
     # Set up the necessary arguments to run the script
+    # For CSD 2023.1
     if sys.platform == 'win32':
         ccdc_coformers_dir = os.path.join(
-            os.path.dirname(io.csd_directory()),
-            'Mercury',
+            io.csd_directory(),
+            os.pardir, os.pardir,
+            'ccdc-software',
+            'mercury',
             'molecular_libraries',
-            'ccdc_coformers'
-        )
-    else:
-        ccdc_coformers_dir = os.path.join(
```

Comment:
For CSD 2023.1 and later, the coformers dir should be the same for all platform?

Reply:
I installed 2023.1 on Linux and the directory layout/naming was the same as Windows. I don't have Mac but assume it would be the same.